File: java-programs/FibnacchiSequenceNth.java

import java.util.Scanner;

public class FibnacchiSequenceNth {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int inputDigit = sc.nextInt();
        int result = findNthofFibnacchiSeries(inputDigit);
        System.out.println(result);
        sc.close();
    }

    public static Integer findNthofFibnacchiSeries(Integer n) {
        int firstDigit = 1;
        int secondDigit = 1;
        int current = 0;
        for (int i = 2; i < n; i++) {
            current = firstDigit + secondDigit;
            firstDigit = secondDigit;
            secondDigit = current;
        }
        return current;
    }
}

File: python-programs/FibnacchiSequenceNth.py
inputDigit = int(input())
firstDigit = 1
secondDigit = 1
start_index = 0;
current = 0
for i in range(start_index + 2, inputDigit):
    current = firstDigit + secondDigit
    firstDigit = secondDigit
    secondDigit = current
    
print(current)
    